app/Community/Policies/TicketPolicy.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Community\Policies;
+
+use App\Community\Models\TriggerTicket;
+use App\Site\Models\Role;
+use App\Site\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class TicketPolicy
+{
+    use HandlesAuthorization;
+
+    public function manage(User $user): bool
+    {
+        return $user->hasAnyRole([
+            Role::DEVELOPER,
```

Comment:
What functionality is "manage" associated to. It seems like other policies use it for moderation. Does this need to be/include `DEVELOPER_STAFF`?

Reply:
`manage` ability checks are used for all Filament resources and in blade views to check for general visibility.
Updated the TicketPolicy to match TriggerTicketPolicy, it's not in use yet.